content/source/docs/plugin/framework/accessing-values.html.md

```diff
@@ -0,0 +1,261 @@
+---
+layout: "extend"
+page_title: "Plugin Development - Framework: Access State, Config, and Plan"
+description: |-
+  How to read values from state, config, and plan in the next-generation
+  Terraform provider development framework.
+---
+
+# Access State, Config, and Plan
+
+There are various points at which the provider needs access to the data from
+the practitioner's configuration, Terraform's state, or the plan that was
+generated. The same patterns are used for accessing this data, regardless of
+its source.
+
+The data is usually stored in a request object:
+
+```go
+func (m myResource) Create(ctx context.Context, req tfsdk.CreateResourceRequest, resp *tfsdk.CreateResourceResponse)
+```
+
+In this example, `req` holds the configuration and plan; there is no state
+value, as the resource doesn't exist in state yet (that's why it's being
+created).
```

Comment:
```suggestion
In this example, `req` holds the configuration and plan, and there is no state
value because the resource does not yet exist in state.
```